docs/TRANSLATING.md

```diff
@@ -246,6 +262,39 @@ git push --force # Push the changes to your repo. Be aware that this will overwr
 
 ---
 
+## Adding a new language (checklist)
+
+The following section describes how to add a new language and what to do or wait for.
+Please open an issue on GitHub and introduce yourself first, even if you intend to add the translation files yourself.
+
+After having opened an issue on GitHub:
+1. Fork the Jamulus repository as described above, then install Qt on your maching. (You will need `lrelease` and `lupdate` later on.)
+2. Create a branch based on the `master` branch: Locally navigate to the folder you cloned the repo with `cd /path/to/jamulus/repo` and type `git checkout -b "translate-add-xx_XX"` where xx_XX is the language code of the new language.
```

Comment:
```suggestion
2. Create a branch based on the `master` branch: Locally navigate to the folder you cloned the repo with (`cd /path/to/jamulus/repo`) and type `git checkout -b "translate-add-xx_XX"` where xx_XX is the language code of the new language.
```

Reply:
Why
> (`cd /path/to/jamulus/repo`)

?  Maybe with "with" inside it would be better:
> (with `cd /path/to/jamulus/repo`)